prow/plank/controller.go

```diff
@@ -24,9 +24,16 @@ import (
 	"time"
 
 	"k8s.io/test-infra/prow/crier"
+	"k8s.io/test-infra/prow/jenkins"
 	"k8s.io/test-infra/prow/kube"
 )
 
+const (
+	guberBasePR   = "https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull"
+	guberBasePush = "https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs"
+	testInfra     = "https://github.com/kubernetes/test-infra/issues"
```

Comment:
testInfraIssue? fileIssueURL? assume this is where to file issues.

Maybe https://github.com/kubernetes/test-infra/issues/new with some template?